capstone/index.yaml

```diff
@@ -6,12 +6,13 @@ indexes:
 
 - kind: event
   properties:
-    - name: availableFood
-    - name: requiredFee
+    - name: university
+    - name: eventTitle
 
 - kind: event
   properties:
     - name: university
+    - name: eventTitle
     - name: rank
       direction: desc
     - name: eventDateTime
```

Comment:
This may cause an error. There is this issue in datastore where your first inequality property needs to be equal to the first sort

Reply:
oh yeah that was just an accidental from fixing merge conflicts!